inc/storefront-template-functions.php

```diff
@@ -165,7 +165,7 @@ function storefront_site_branding() {
 			jetpack_the_site_logo();
 		} else { ?>
 			<div class="site-branding">
-				<h1 class="site-title"><a href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home"><?php bloginfo( 'name' ); ?></a></h1>
+				<div class="beta site-title"><a href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home"><?php bloginfo( 'name' ); ?></a></div>
```

Comment:
About logo or branding, SEO plugin such as Yoast already take care of many things. Optionally, we could use shema.org for the item "site" on every pages. Check this method on my WooCommerce PR.

```php
public function generate_shop_data() {
 +    if ( ! is_shop() || ! is_front_page() ) {
 +      return;
 +    }
 +
 +    $json['@type']             = 'WebSite';
 +    $json['name']              = get_bloginfo( 'name' );
 +    $json['url']               = get_bloginfo( 'url' );
 +    $json['potentialAction'] = array(
 +      '@type'                => 'SearchAction',
 +      'target'               => get_bloginfo( 'url' ) . '/?s={search_term_string}&post_type=product',
 +      'query-input'          => 'required name=search_term_string'
 +    );
 +
 +    $this->set_data( apply_filters( 'woocommerce_structured_data_shop', $json ) );
 +  }
 +}